src/target/riscv/riscv-013.c

```diff
@@ -3811,13 +3826,18 @@ write_memory_abstract(struct target *target, const riscv_mem_access_args_t args)
 
 	RISCV013_INFO(info);
 	int result = ERROR_OK;
-	bool use_aampostincrement = info->has_aampostincrement != YNM_NO;
 
 	/* Convert the size (bytes) to width (bits) */
 	unsigned int width = args.size << 3;
 
 	/* Create the command (physical address, postincrement, write) */
-	uint32_t command = access_memory_command(target, false, width, use_aampostincrement, true);
+	uint32_t command = access_memory_command(target, false, width,
+			/* postincrement = */ true, /* is_write = */ true);
```

Comment:
1) There is a typo: `is_write` must be false.

```suggestion
			/* postincrement = */ true, /* is_write = */ false);
```

2) Could you please check if abstract memory commands are in fact tested in the CI?

3) This part of code can be simplified (the `if` is redundant) - same as mentioned above.

Reply:
Addressed